portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase

```diff
@@ -119,773 +140,945 @@ definition {
+		task ("When: User enables propagation of the template") {
+			SiteTemplatesNavigator.gotoSiteTemplatesPage(
+				pageName = "Home",
+				siteName = "${siteName}",
+				siteTemplateName = "Site Template Name");
 
-		SiteTemplatesNavigator.gotoSiteTemplatesPage(
-			pageName = "Home",
-			siteName = "${siteName}",
-			siteTemplateName = "Site Template Name");
+			SiteTemplates.enablePropagationPG();
+		}
 
-		SiteTemplates.enablePropagationPG();
+		task ("Then: The portlet will appear on the new site") {
+			Navigator.openSiteURL(siteName = "Site Name");
 
-		Navigator.openSiteURL(siteName = "Site Name");
+			Refresh();
 
-		Refresh();
+			AssertElementPresent(
+				key_portletName = "AssetPublisher",
+				locator1 = "Portlet#SPECIFIC_FRAGMENTS_AND_WIDGETS");
+		}
 
-		AssertElementPresent(
-			key_portletName = "AssetPublisher",
-			locator1 = "Portlet#SPECIFIC_FRAGMENTS_AND_WIDGETS");
 	}
 
 	@description = "This is a use case for LPS-133675. Javascript fields should be propagated on site templates."
 	@priority = "3"
 	test ViewJavaScriptPropagateviaSiteTemplate {
-		var javascript = '''alert("alert!");''';
+			var javascript = '''alert("alert!");''';
 
-		SiteTemplates.addCP(siteTemplateName = "Site Template Name");
-
-		SiteTemplates.gotoSettingsCP(siteTemplateName = "Site Template Name");
-
-		SiteTemplates.AddSiteTemplateJavaScript(javaScript = "${javascript}");
+		task ("Given: User adds new site template") {
+			SiteTemplates.addCP(siteTemplateName = "Site Template Name");
+		}
 
-		ProductMenu.gotoPortlet(
-			category = "Site Builder",
-			portlet = "Pages");
+		task ("When: User adds javascript code to the site") {
+			SiteTemplates.gotoSettingsCP(siteTemplateName = "Site Template Name");
 
-		PagesAdmin.gotoPageEllipsisMenuItem(
-			menuItem = "View",
-			pageName = "Home");
+			SiteTemplates.AddSiteTemplateJavaScript(javaScript = "${javascript}");
+		}
 
-		AssertConfirm(value1 = "alert!");
+		task ("Then: The script should be escaped in on the home page of th site template") {
+			ProductMenu.gotoPortlet(
+				category = "Site Builder",
+				portlet = "Pages");
 
-		ApplicationsMenu.gotoPortlet(
-			category = "Sites",
-			panel = "Control Panel",
-			portlet = "Sites");
+			PagesAdmin.gotoPageEllipsisMenuItem(
+				menuItem = "View",
+				pageName = "Home");
 
-		Site.addSiteTemplateCP(
-			siteName = "Site Name",
-			siteTemplateName = "Site Template Name");
+			AssertConfirm(value1 = "alert!");
+		}
 
-		SiteTemplates.ViewSiteTemplateJavaScript(javaScript = "${javascript}");
-	}
+		task ("When: User adds a new site based on the site template") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Sites",
+				panel = "Control Panel",
+				portlet = "Sites");
 
+			Site.addSiteTemplateCP(
+				siteName = "Site Name",
+				siteTemplateName = "Site Template Name");
+		}
+	
+		task ("Then: The javascript will be inherited from the site template") {
+			SiteTemplates.ViewSiteTemplateJavaScript(javaScript = "${javascript}");
+		}
```

Comment:
@weller0246 One closing bracket is missing here, so now the last test case is open.

Reply:
I added it back